lgc/util/GfxRegHandler.cpp

```diff
@@ -160,6 +163,24 @@ void SqImgSampRegHandler::setReg(SqSampRegs regId, Value *regValue) {
   }
 }
 
+// =====================================================================================================================
+// SqImgSampReg Bits infomation look up table (Gfx8)
+// Refer to imported/chip/gfx6/si_ci_merged_registers.h : SQ_IMG_RSRC_WORD
+static constexpr BitsInfo SqImgRsrcRegBitsGfx8[static_cast<unsigned>(SqRsrcRegs::Count)] = {
```

Comment:
SqImgRsrcRegBitsGfx6 should be better since we're referencing gfx6/si_ci_merged_registers.h